src/main/java/com/revature/registry/model/Organization.java

```diff
@@ -0,0 +1,30 @@
+package com.revature.registry.model;
+
+import lombok.Data;
+
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * An {@link Organization} refers to a Github Organization.
+ *
+ * Typically, the root/first {@link Iteration} Github Repository will be found in the `revaturelabs` Organization.
+ * Subsequent sprints have an individual {@link Organization} created, with as many {@link Repository}'s
+ * as required to meet the applications needs.
+ */
+
+@Entity
+@Data
+public class Organization {

```

Comment:
Should Organization have a bi-directional mapping with Repository?

Reply:
Yes, I believe it should! Will include in next commit.